src/libs/auth.js

```diff
@@ -45,6 +45,16 @@ export const initializeAuth = _.memoize(async () => {
       const authResponse = user.getAuthResponse(true)
       const profile = user.getBasicProfile()
       const isSignedIn = user.isSignedIn()
+      //The following few lines of code are to handle sign-in failures due to privacy tools.
+      if (state.isSignedIn===false && isSignedIn === false) {
+        //if both of these values are false, it means that the user was initially not signed in (state.isSignedIn === false),
+        //tried to sign in (invoking processUser) and was still not signed in (isSignedIn === false).
+        notify('info', 'Having trouble logging in?', {
```

Comment:
```suggestion
        notify('error', 'Could not sign in', {
```
Should this be "error" to clarify that a user's action was not successful?  

Also, it feels weird to click a "Sign in", select Google account, and then the next thing you see is this message. I didn't know I wasn't signed in or couldn't be, until I clicked the details. I recommend changing the heading to make it clearer that a user hasn't been signed in.

Reply:
Good point! This is more clear